examples/workflows/06-cdb-to-pymechanical-workflow.py

```diff
@@ -0,0 +1,380 @@
+# used to postprocess the simulation.
+engineering_constants = (
+    pyacp.material_property_sets.ConstantEngineeringConstants.from_orthotropic_constants(
+        E1=5e10, E2=1e10, E3=1e10, nu12=0.28, nu13=0.28, nu23=0.3, G12=5e9, G23=4e9, G31=4e9
+    )
+)
+
+strain_limit = 0.01
+strain_limits = pyacp.material_property_sets.ConstantStrainLimits.from_orthotropic_constants(
+    eXc=-strain_limit,
+    eYc=-strain_limit,
+    eZc=-strain_limit,
+    eXt=strain_limit,
+    eYt=strain_limit,
+    eZt=strain_limit,
+    eSxy=strain_limit,
+    eSyz=strain_limit,
+    eSxz=strain_limit,
+)
+
+ud_material = model.create_material(
+    name="UD",
+    ply_type=pyacp.PlyType.REGULAR,
+    engineering_constants=engineering_constants,
+    strain_limits=strain_limits,
+)
+
+fabric = model.create_fabric(name="UD", material=ud_material, thickness=0.1)
+
+
+# %%
+# Define a rosette and oriented selection set. Plot the orientation.
+rosette = model.create_rosette(origin=(0.0, 0.0, 0.0), dir1=(1.0, 0.0, 0.0), dir2=(0.0, 0.0, 1.0))
+
+oss = model.create_oriented_selection_set(
+    name="oss",
+    orientation_point=(0.0, 0.0, 0.0),
+    orientation_direction=(0.0, 1.0, 0),
+    element_sets=[model.element_sets["All_Elements"]],
+    rosettes=[rosette],
+)
+
+model.update()
+
+plotter = pyacp.get_directions_plotter(model=model, components=[oss.elemental_data.orientation])
+plotter.show()
+
+
+# %%
+# Create various plies with different angles and add them to a modeling group.
+modeling_group = model.create_modeling_group(name="modeling_group")
+angles = [0, 45, -45, 45, -45, 0]
+for idx, angle in enumerate(angles):
+    modeling_group.create_modeling_ply(
+        name=f"ply_{idx}_{angle}_{fabric.name}",
+        ply_angle=angle,
+        ply_material=fabric,
+        oriented_selection_sets=[oss],
+    )
+
+model.update()
+
+
+# %%
+# Show the fiber directions of a specific ply.
+modeling_ply = model.modeling_groups["modeling_group"].modeling_plies["ply_4_-45_UD"]
+
+
+fiber_direction = modeling_ply.elemental_data.fiber_direction
+assert fiber_direction is not None
+plotter = pyacp.get_directions_plotter(
+    model=model,
+    components=[fiber_direction],
+)
+
+plotter.show()
+
+
+# %%
+# For a quick overview, print the model tree. Note that
+# the model can also be opened in the ACP GUI. For more
+# information, see :ref:`view_the_model_in_the_acp_gui`.
+pyacp.print_model(model)
+
+# %%
+# Save the ACP model
+# ------------------
+
+composite_definitions_h5_filename = "ACPCompositeDefinitions.h5"
+matml_filename = "materials.xml"
+
+model.export_shell_composite_definitions(working_dir_path / composite_definitions_h5_filename)
+model.export_materials(working_dir_path / matml_filename)
+
+
+# %%
+# Import mesh, materials and plies into Mechanical
+# ------------------------------------------------
+#
+# Import geometry, mesh, and named selections into Mechanical
+
+
+# TODO: rename 'import_acp_solid_mesh'? Here it is used with a shell mesh.
```

Comment:
To discuss.

Reply:
Yes, the function load a CDB into Mechanical and sets some key options specific to layered elements. 
what about `import_acp_mesh_from_cdb`?

Reply:
How about `import_layered_element_mesh`? In this example we are actually loading the original `.dat` file that was also loaded into ACP.
I guess we _could_ instead use the CDB exported from ACP, not sure if that's supposed to make a difference.

Reply:
I would limit it to CDBs (cdb, dat) generated by ACP only for the moment. Other formats might be added in the future such as k (LS Dyna input file) but the current helper function contains some CDB specific options such as "check valid block format" which we disable.